.infrastructure/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-todoapp
+  namespace: todoapp
```

Comment:
The `namespace` field should be removed from the PersistentVolume configuration. PersistentVolumes are cluster-wide resources and do not belong to any specific namespace.